lib/coverband/utils/results.rb

```diff
@@ -20,6 +20,31 @@ def file_with_type(source_file, results_type)
         get_results(results_type).source_files.find { |file| file.filename == source_file.filename }
       end
 
+      def runtime_relevant_coverage(source_file)
+        return unless eager_loading_coverage && runtime_coverage
+
+        eager_file = get_eager_file(source_file)
+        runtime_file = get_runtime_file(source_file)
+
+        return 0.0 unless runtime_file
+
+        return runtime_file.formatted_covered_percent unless eager_file
+
+        runtime_relavant_lines = eager_file.relevant_lines - eager_file.covered_lines_count
+        runtime_file.runtime_relavant_calculations(runtime_relavant_lines) { |file| file.formatted_covered_percent }
+      end
+
+      def runtime_relavent_lines(source_file)
+        return 0 unless eager_loading_coverage && runtime_coverage
```

Comment:
Would it be possible that a file is only accessed at runtime? In this case would we just return runtime_coverage?

Reply:
yeah see line 43, this just expects that we will get eager and runtime coverage reports, which we should always expect so long as it has been initalized, but we check for the specific files down below.

https://github.com/danmayer/coverband/pull/270/files#diff-853e3c9a7d0f6b56a991f069a252d0cfR43

Reply:
I see. Although probably unlikely I feel like eager_loading_coverage could be nil if we are not in a rails project or using bundler. Assuming that is true, In this case, would we want to return `runtime_file.covered_lines_count`?

Reply:
OK, I thought how we save it would still have an empty report not nil, but I updated to use runtime regardless.